kafka/source/pkg/apis/sources/v1alpha1/kafka_conversion_test.go

```diff
@@ -163,6 +163,9 @@ func TestKafkaSourceConversionRoundTripV1alpha1(t *testing.T) {
 						Conditions: duckv1.Conditions{{
 							Type:   "Ready",
 							Status: "True",
+						}, {
+							Type:   "SinkProvided",
+							Status: "True",
```

Comment:
When I added this condition, the test started to fail with the previous conversion code.